src/assets/js/preview-environment/components/test-component.js

```diff
@@ -0,0 +1,18 @@
+export default ({  _editable, title, text }) => {
+  /**
+   * @TODO: use _editable to prefix the component html
+   * with so we can have full live preview thingies by
+   * clicking on the component itself
+   *
+   * Kris: "Maybe we can use _editable in the forEach when
+   * rendering the component and just say innerHTML += editablecomment
+   * before += componentHTML"
+   */
+
+  return `
```

Comment:
Let Nunjucks generate the HTML template. Make a reference to the component library so that we don't have to create double the components (JS & Nunjucks). Easier to use and DRY.

Reply:
I'll try to implement this as a follow up to this review so we just have it handled in a nice way. As I see it I'll just copy the templates on build time to some kind of `views` so the development environment can have access to it. Anyway, will fix it here.

Reply:
@gijslaarman I tried to fix this using Nunjucks but it gets more of a hassle so I don't think it's worth implementing it for MVP. Maybe we can fix it now by writing better docs and come up with a better solution later. Could you merge this PR as is? 🥺